docker/app/install-composer.sh

```diff
@@ -4,7 +4,7 @@
 # Origin: https://github.com/composer/composer/blob/master/doc/faqs/how-to-install-composer-programmatically.md
 # Licence: MIT
 
-EXPECTED_SIGNATURE=$(wget -q -O - https://composer.github.io/installer.sig)
```

Comment:
> It reduces the complexity of the build process and the maintenance.

Not really. `wget` is already provided by BusyBox. Installing `openssl` is required for HTTPS support. And that's it. I don't see where the complexity lies.

> As pointed out by Jordi, there is no point using the old method when using PHP 7+. And we're using PHP 7+.

Agreed 100%. But we don't really gain anything by changing it. It only creates another maintenance issue, if anything. (Instead of just copy-and-paste from the official sample script, now we have our own modifications. :laughing:)